pkg/cmd/ssh/command.go

```diff
@@ -194,6 +280,11 @@ func runCommand(f util.Factory, o *Options) error {
 				_ = os.Remove(o.SSHPublicKeyFile)
 				_ = os.Remove(o.SSHPrivateKeyFile)
 			}
+
+			// though technically not used _on_ the bastion itself, without
+			// this file remaining, the user would not be able to use the SSH
+			// command we provided to connect to the shoot nodes
+			defer os.Remove(nodePrivateKeyFile)
```

Comment:
does it make sense to print out the left-overs that are not cleaned up in case keepBastion is true, so that the user has a chance to delete / reuse them? WDYT

Reply:
Can do

Reply:
sorry, what I meant was to handle the `else`-case
```go
if !o.KeepBastion {
    // ..
} else {
  // print out left-overs that are not cleaned
}